.mergify.yml

```diff
@@ -1,14 +1,6 @@
 queue_rules:
-  - name: default
-    conditions:
-      # Conditions to get out of the queue (= merged)
-      - status-success=Run Unit Tests
-      - status-success=Semantic Pull Request
-      - status-success=Analyze (javascript)
-
-pull_request_rules:
-  - name: Automatically merge on CI success and review approval
-    conditions:
+  - name: duplicated default from Automatically merge on CI success and review approval
+    queue_conditions:
```

Comment:
`queue_rules->queue_conditions` logic is duplicated from the`pull_request_rules -> conditions` for the condition named "for "Automatically merge on CI success and review approval"